src/acl.c

```diff
@@ -2556,6 +2556,61 @@ int aclAddReplySelectorDescription(client *c, aclSelector *s) {
     return 3;
 }
 
+sds addACLsToUser(user *u, sds username, sds *argv, int argc) {
```

Comment:
This method should be part of the acl api, as `ACLSetUser()`

Reply:
just to note renamed to ACLStringSetUser